docs/releases/2.23.0.md

```diff
@@ -0,0 +1,27 @@
+#### Release Links
+* lagoon [v2.23.0](https://github.com/uselagoon/lagoon/releases/tag/v2.23.0)
+* lagoon-ui [core-v2.23.0](https://github.com/uselagoon/lagoon-ui/releases/tag/core-v2.23.0)
+* lagoon-build-deploy [core-v2.23.0](https://github.com/uselagoon/build-deploy-tool/releases/tag/core-v2.23.0)
+* lagoon-core chart [1.50.0](https://github.com/uselagoon/lagoon-charts/releases/tag/lagoon-core-1.50.0)
+* lagoon-remote chart [0.96.0](https://github.com/uselagoon/lagoon-charts/releases/tag/lagoon-remote-0.96.0)
+
+## Release Notes
+
+This release is a standard Lagoon release. There are changes that teams managing their own Lagoon installations (core and remote) may need to be aware of.
+
+## Upgrades
+
+Self-managed Lagoon installs may need to be aware of these changes in lagoon-core.
+
+### Broker update to RabbitMQ 3.13
+This release of Lagoon includes an update to RabbitMQ 3.13 in https://github.com/uselagoon/lagoon/pull/3849 - which has necessitated a small modification to the feature flag enabler (to ignore experimental plugins like Khepri, that are known to have issues). As long as the upgrade runs in order successfully, the plugin should remain disabled.
+
+### KEYCLOAK_ADMIN_API_CLIENT_SECRET
+In https://github.com/uselagoon/lagoon/pull/3841 a new client was added to the Master realm in Keycloak to allow the API to configure Keycloak. This will remove the need for an admin user/password to be used, so MFA can be set on the admin account. The charts will autocreate and configure the secret if none is provided.
```

Comment:
Might be worth mentioning to not set MFA or other on the admin account until AFTER the upgrade? Since doing so before the upgrade will result in a failure.